external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -29,45 +30,36 @@ if(${ENABLE_PCMSolver})
                        -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
                        -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DPYMOD_INSTALL_LIBDIR=${PYMOD_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
                        -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DENABLE_psi4=ON
```

Comment:
done on the PCMSolver PR side. I'll let you make whatever HostWriter changes you want to this PR before editing it again and patching up this and the other things you mention below.